ci/ansible/roles/pulp-crane/tasks/main.yml

```diff
@@ -30,9 +43,33 @@
     group: root
     mode: 0644
```

Comment:
This "Install Crane configuration file" task should now have a `when:` clause. Without such a clause, this task will always execute. On RHEL 6, the "Install Crane configuration file (RHEL 6)" task will also execute, causing two unnecessary modifications on every run.